ui/src/environments/environment-type.ts

```diff
@@ -43,7 +43,6 @@ export interface Environment {
   // The base URL for the Zendesk help center / user forum.
   // Example value: https://aousupporthelp.zendesk.com/hc/
   zendeskHelpCenterUrl: string;
```

Comment:
Rather than trying to parse this URL and implicitly identify the Zendesk environment we're binding to, it may be simpler (and more true to the intent) to have 'prod' and 'sandbox' as the allowable env variables here, and use that to key the `urls` map above.

Reply:
Good suggestion. Done. I think this way is clearer. I also moved the widget key out, since it follows from the environment type.